tests/nimble_rpble_gnrc_ext/Makefile

```diff
@@ -0,0 +1,28 @@
+BOARD ?= nrf52dk
+include ../Makefile.tests_common
+
+# include shell support
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Include GNRC and RPL
+USEMODULE += gnrc_netif_single
```

Comment:
Why is this needed?

Reply:
To my understanding, this triggers the auto-initialization for RPL, which won't work without it. Am I wrong here?!

Reply:
AFAIK this is about optimization when there is only one network interface onboard, unrelated to auto-initialization. I guess for a tests its not needed, but might help the test be compiled for more BOARDs